examples/jsm/controls/DragControls.js

```diff
@@ -75,17 +84,31 @@ class DragControls extends EventDispatcher {
 			if ( scope.enabled === false ) return;
 
 			updatePointer( event );
-
+			_end.copy(_pointer);
+			_diff.subVectors(_end, _start);
+			_start.copy(_end);
+			
 			_raycaster.setFromCamera( _pointer, _camera );
-
+			
 			if ( _selected ) {
+				
+				if (scope.mode === "translate") {
+				
+					if ( _raycaster.ray.intersectPlane( _plane, _intersection ) ) {
+						
+						_selected.position.copy( _intersection.sub( _offset ).applyMatrix4( _inverseMatrix ) );
+						
+					}
+				
+				} else if (scope.mode === "rotate") {
 
-				if ( _raycaster.ray.intersectPlane( _plane, _intersection ) ) {
-
-					_selected.position.copy( _intersection.sub( _offset ).applyMatrix4( _inverseMatrix ) );
-
+					_diff.multiplyScalar(2); //Just a random scalling factor since it rotated too slow
```

Comment:
Instead of using a fixed number, let's add a `rotateSpeed` property like in `OrbitControls` with `1` as the default value.